packages/manager/apps/pci-rancher/src/components/layout-helpers/CreateRancher/CreateRancher.tsx

```diff
@@ -0,0 +1,244 @@
+  };
+
+  const onCancelClick = () =>
+    navigate(`/pci/projects/${projectId}/rancher/onboarding`);
+
+  return (
+    <div className="max-w-3xl">
+      <Title>{t('createRancherTitle')}</Title>
+      <OsdsMessage
+        color={ODS_THEME_COLOR_INTENT.info}
+        type={ODS_MESSAGE_TYPE.info}
+        className="my-6"
+      >
+        <OsdsText color={ODS_THEME_COLOR_INTENT.text}>
+          {t('createRancherInfoMessage')} <br />
+        </OsdsText>
+      </OsdsMessage>
+      {hasRancherCreationError && (
+        <OsdsMessage
+          color={ODS_THEME_COLOR_INTENT.error}
+          type={ODS_MESSAGE_TYPE.error}
+          className="my-6"
+        >
+          <OsdsText color={ODS_THEME_COLOR_INTENT.text}>
+            {t('createRancherError')} <br />
+          </OsdsText>
+        </OsdsMessage>
+      )}
+      <div>
+        <div className="mb-8">
+          <Block>
+            <Subtitle>{t('createRancherName')}</Subtitle>
+          </Block>
+          <Block>
+            <OsdsText color={ODS_THEME_COLOR_INTENT.text}>
+              {t('createRancherDescription')}
+            </OsdsText>
+          </Block>
+          <OsdsInput
+            aria-label="rancher-name-input"
+            type={ODS_INPUT_TYPE.text}
+            color={
+              hasInputError
+                ? ODS_THEME_COLOR_INTENT.error
+                : ODS_THEME_COLOR_INTENT.info
+            }
+            className="my-3 w-1/3"
+            value={rancherName}
+            onOdsValueChange={(
+              e: OsdsInputCustomEvent<OdsInputValueChangeEventDetail>,
+            ) => setRancherName(e.target.value as string)}
+          />
+        </div>
+
+        <div className="my-3">
+          <Subtitle>{t('createRancherServiceLevel')}</Subtitle>
+        </div>
+        <Block>
+          <OsdsText color={ODS_THEME_COLOR_INTENT.text}>
+            {t('createRancherServiceLevelTitle')}
+          </OsdsText>
+        </Block>
+        <OsdsText color={ODS_THEME_COLOR_INTENT.text}>
+          {t('createRancherServiceLevelDescription')}
+        </OsdsText>
+        <div className="flex my-5">
+          {plans?.map((v) => (
+            <TileSection
+              isActive={v.name === activePlan?.name}
+              name={t(v.name)}
+              description={t(getRancherPlanDescription(v.name))}
+              chipLabel={v.name === 'OVHCLOUD_EDITION' ? t('comingSoon') : ''}
+            />
+          ))}
+        </div>
+
+        <Block>
+          <Subtitle>{t('createRancherVersion')}</Subtitle>
+        </Block>
+        <Block>
+          <OsdsText color={ODS_THEME_COLOR_INTENT.text}>
+            {t('createRancherVersionTitle')}
+          </OsdsText>
+        </Block>
+        <Block>
+          <OsdsText color={ODS_THEME_COLOR_INTENT.text}>
+            {t('createRancherVersionDescription')}
+          </OsdsText>
+        </Block>
+        <div className="flex my-5">
+          {versions?.map((v) => (
+            <TileSection
```

Comment:
You should use the radio + tile like vrackservices
packages/manager/apps/vrack-services/src/pages/create/components/RegionFormField.tsx